packages/vue-client/src/components/GamesFilterForm.vue

```diff
@@ -0,0 +1,61 @@
+<script setup lang="ts">
+import { useCurrentUser } from "@/stores/user";
+import {
+  getVariantList,
+  type GamesFilter,
+} from "@ogfcommunity/variants-shared";
+import { type Ref, ref, computed } from "vue";
+
+const emit = defineEmits<{ (e: "filterChange", filter: GamesFilter): void }>();
+function emitFilter() {
+  emit("filterChange", filter.value);
+}
+
+const variants: string[] = getVariantList();
+const selectedVariant: Ref<string | ""> = ref("");
```

Comment:
Why `string | ""`? Isn't `""` a `string` anyway?

Reply:
Yeah 😅 ... I used to have string | null here, consistently with the type in the GamesFilter object. But somehow I failed at making an <option> with value = null, so I decided to use "" instead. I left it in the type explicitly like this, to emphasize that "" has a special meaning, but I'm unhappy about the whole construction here.

Reply:
I see, looks like `null` is indeed not possible.